### DISCUSSION TITLE
Conflicting info regarding need for exchange/Office/outlook.com

### DISCUSSION THREAD

**Discussion by BKLTMS** [YetiForceCompany/YetiForceCRM#17162] [General]
1) In link to Outlook Mail Integration website from Marketplace it states : Does not require Exchange or Microsoft 365 connections.
     And that it will work with the Outlook Desktop Client.

2) In the online Outlook Add-in documentation it states Danger the above are needed and that IMAP or POP3 accounts will not work.

Question:  Which of the above is accurate/correct? 

Thanks in advance!

**Comment by mariuszkrzaczkowski** [comment]
here are the current requirements
https://doc.yetiforce.com/administrator-guides/integration/mail-integration-panel/outlook/#requirements

**Comment by BKLTMS** [comment]
Thanks for the response.

Then a standalone Outlook as part of an MS Office install will not work?

The docs through the link provided above indicate that an Exchange or Office 365 connection is necessary.  A suggestion then, conforming the information on the website link, contained in the marketplace within the app, as it states the exact opposite.

![Outlook Yeti Marketplace Website_Screenshot 2023-06-23 074627](https://github.com/YetiForceCompany/YetiForceCRM/assets/137448162/fcb2cb3b-da34-4a2a-98d9-6b00ac38ad27)

**Reply by mariuszkrzaczkowski** [reply]
where did you find this description?

**Comment by BKLTMS** [comment]
Below is the link from the Product Preview "Website" button when addon is clicked in Marketplace within the App.
https://yetiforce.com/pl/sklep/1-integracje/1-yetiforce-outlook-integration-panel.html

The referenced in red FAQ section at the bottom of the page really does not make it clear either.

Hope this helps.
